assets/src/edit-story/components/panels/stylePreset/test/presetGroup.js

```diff
@@ -57,11 +57,11 @@ const getThirdArguments = (calls) => {
 
 describe('stylePresets/PresetGroup', () => {
   it('should modify active index correctly with arrow keys', () => {
-    const { itemRenderer, getByText } = setupPresetGroup();
+    const { itemRenderer, getAllByRole } = setupPresetGroup();
     expect(getThirdArguments(itemRenderer.mock.calls)).toStrictEqual([0, 0, 0]);
 
-    const label = getByText('Colors');
-    const group = label.nextSibling;
+    const firstButton = getAllByRole('button')[0];
+    const group = firstButton.parentNode;
```

Comment:
Hm, maybe a better way to get this would be to put a `role="listbox"` on the group?

Reply:
And this group could even have a `aria-labelledby` pointing to the label element (_"Colors"_) before it making the query even cleaner.

Reply:
"Colors" label was removed (for now) so can't really label it by it anymore unfortunately. About the listbox -- in this case the buttons would need to turn to options which also doesn't seem really accurate.

Perhaps we can leave it as-is until the final UX gets resolved for the presets?

Reply:
Sure, but maybe put a comment in somewhere with a todo.